src/lib/components/Game.svelte

```diff
@@ -132,11 +136,8 @@
     updatePlayerTurnAndText(prevPlayer);
   }
 
-  function updateMovesList(event) {
-    if (movesList.currentListItem.children.length === 2) {
-      movesList.nextListItem();
-    }
-    movesList.addMove(event.detail.move.toString());
+  function updateMovesList(event: MouseEvent) {
```

Comment:
move would be a more natural parameter here.
```suggestion
  function updateMovesList(move: Move) {
```